src/python/pants/backend/go/util_rules/BUILD

```diff
@@ -1,5 +1,10 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_library()
-python_tests(name='tests')
+python_library(dependencies=[":analyzer_sources"])
+python_tests(name='tests', dependencies=[":analyzer_sources"])
```

Comment:
No need to add to `dependencies` because it's added to the `python_library`

Reply:
Done